bugbug/models/assignee.py

```diff
@@ -105,11 +105,19 @@ def get_labels(self):
         for assignee, count in assignee_counts:
             print(f"{assignee}: {count}")
 
-        return {
-            bug_id: assignee
-            for bug_id, assignee in classes.items()
-            if assignee in top_assignees
-        }
+        return (
+            {
+                bug_id: assignee
+                for bug_id, assignee in classes.items()
+                if assignee in top_assignees
+            },
+            sorted(
```

Comment:
Let's move the sorting in model.py, so we don't have to replicate the code in each model.